.circleci/config.yml

```diff
@@ -1552,7 +1552,7 @@ jobs:
       OPTA_DISABLE_REPORTING: "true"
     steps:
       - checkout-opta
-      - install-python
+      # - install-python
```

Comment:
```suggestion
```

If python isn't needed, the line should be removed entirely instead of commented out. Also, this change seems out of scope for this PR and could be done separately. (Ditto for the equivalent changes later in the file)

Reply:
This was removed - Nilesh did the circleci changes in another PR